packages/storybook/stories/va-file-input-uswds.stories.tsx

```diff
@@ -180,7 +182,7 @@ export const HeaderLabel = Template.bind(null);
 HeaderLabel.args = {
   ...defaultArgs,
   label: 'Header label',
-  headerSize: 3,
+  'header-size': 3,
```

Comment:
> [!note>
> Flagging this change for awareness because it seemed to be an outcome of introducing `...rest` into the template. It seemed to already exist in a few component stories, but just wanting confirmation that `rest` won't have unintended consequences here.